plugins/module_utils/network/ios/argspec/bgp_global/bgp_global.py

```diff
@@ -349,6 +349,9 @@ class Bgp_globalArgs(object):  # pylint: disable=R0903
                 "neighbor": {
                     "type": "list",
                     "elements": "dict",
+                    "mutually_exclusive": [
```

Comment:
Ansible won't recognize the `mutually_exclusive` key here as it is intended to work, rather it is recommended to use `choice` to handle the same. This is supposed to break parts of the execution.